src/rulesets/operations.ts

```diff
@@ -169,6 +169,7 @@ export const rules = {
     request.queryParameter.added.must(
       "not be required",
       (queryParameter, context, docs) => {
+        if (context.operationAdded) return;
```

Comment:
for rules about required, they can be required only if the operation was also added at the same time (there to start).